test/unit/org/apache/cassandra/tools/nodetool/stats/TableStatsPrinterTest.java

```diff
@@ -70,6 +70,14 @@ public class TableStatsPrinterTest extends TableStatsTestBase
         "\tAverage tombstones per slice (last five minutes): 5.00\n" +
         "\tMaximum tombstones per slice (last five minutes): 1\n" +
         "\tDroppable tombstone ratio: 0.000\n" +
+        "\tSAI local query latency (mean): 0.000 ms\n" +
+        "\tSAI post-filtering latency (mean): 0.000 ms\n" +
+        "\tSAI space used (bytes): null\n" +
+        "\tSAI SSTable indexes hit per query (mean): 0.0\n" +
+        "\tSAI index segments hit per query (mean): 0.0\n" +
+        "\tSAI rows filtered per query (mean): 0.0\n" +
+        "\tSAI local total query timeouts: 0\n" +
```

Comment:
nit: I guess we could just make this "local query timeouts" and leave out the "total". It's fairly obvious this is a count and not a rate...

(see also below if you do change it)

Reply:
Corrected the name.